.github/workflows/container-build.yml

```diff
@@ -59,5 +59,21 @@ jobs:
       - name: creds
         run: gcloud auth configure-docker --quiet
 
+      - name: Formatted labels
+        id: labels
+        run: |
+          FORMATED_LABELS="--image-label commit-hash=$GITHUB_SHA"
+
+          BRANCH_NUMBER=$(gh pr list --state all --search "sha:$GITHUB_SHA" --label "breaking-change" | awk '{print $1}')
+          echo "Branch Number: $BRANCH_NUMBER" 
+
+          # Check if a pull request number was found
+          if [ -n "$BRANCH_NUMBER" ]; then
+            FORMATED_LABELS+=" --image-label breaking-change=true"
+          fi
+          echo "FORMATED_LABELS='$FORMATED_LABELS'" >> $GITHUB_OUTPUT
+
       - name: container
-        run: KO_PREFIX=gcr.io/projectsigstore/fulcio/ci/fulcio make sign-keyless-ci
+        run: |
+          echo "Formated Label: ${{ steps.labels.outputs.FORMATED_LABELS }}"
```

Comment:
We could tackle this in the next PR, but I think we also need to handle labeling containers with merge times such that we can sort them and deploy only the latest commit. Otherwise if we merge two PRs at the same time, and the earlier GitHub action completes after the later GitHub action, then we would deploy an older commit.

Reply:
It would be good. but we can also look to the repo's commit history and check the order, right?

Reply:
It already addressed. We can check it in the deployment side, checking the tags or commits that were created after the last deployment, and check that any of them has or not the`breaking-change` label.

Reply:
We are adding in this same PR a new tag for the containers that will help us to address it, allowing us to sort them correctly